BasicController.java

```diff
@@ -25,7 +25,7 @@ public ApiResult add(@RequestParam BigDecimal augend, @RequestParam BigDecimal a
 
    @GetMapping("/subtract")
    public ApiResult subtract(@RequestParam BigDecimal minuent, @RequestParam BigDecimal subtrahend) {
-      return getResult(minuent.subtract(subtrahend));
+      return (minuent.subtract(subtrahend));
```

Comment:
The return statement in the `subtract` method has been changed to return a `BigDecimal` directly instead of wrapping it in an `ApiResult` object. This will cause a type mismatch error because the method signature expects an `ApiResult` return type.